### PR TITLE
Add note about how to create model mixins

### PR DESCRIPTION
**Pull request by codingjoe**

### CODE REVIEW

File: heroku_connect/db/models/base.py [hunk 1]
```diff
@@ -60,6 +60,12 @@ class MyCustomObject(hc_models.HerokuConnectModel):
         on :attr:`settings.HEROKU_CONNECT_SCHEMA<.HerokuConnectAppConf.HEROKU_CONNECT_SCHEMA>`
         and :attr:`.sf_object_name`.
 
+    Note:
+
+        A model mixin must inherit from :class:`Meta.managed<django.db.models.Model>`
+        not `.HerokuConnectModel`. Only the final (not abstract) models should inherit
+        from `.HerokuConnectModel` otherwise build in fields will clash.
```
Comment (iamkhush) [inline heroku_connect/db/models/base.py]: I think you want the last sentence to be - `otherwise the built-in fields will clash`

Comment (codingjoe) [reply heroku_connect/db/models/base.py]: better?

File: heroku_connect/db/models/base.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ class MyCustomObject(hc_models.HerokuConnectModel):
 
         A model mixin must inherit from :class:`Meta.managed<django.db.models.Model>`
         not `.HerokuConnectModel`. Only the final (not abstract) models should inherit
-        from `.HerokuConnectModel` otherwise build in fields will clash.
+        from `.HerokuConnectModel` otherwise build-in fields will clash.
```
Comment (iamkhush) [inline heroku_connect/db/models/base.py]: Although a small change - I wanted to imply that its `built-in` . 
https://docs.djangoproject.com/en/2.0/howto/custom-model-fields/#emulating-built-in-field-types

Comment (codingjoe) [reply heroku_connect/db/models/base.py]: I fail to see how this article is related. I am talking about the fields build into `HerokuConnectModel`.

Comment (iamkhush) [reply heroku_connect/db/models/base.py]: Yes, I understand. The link was just to show the heading, which reads `built-in fields`  and not `build-in`.

**Pull request merged**